Makefile

```diff
@@ -112,6 +112,10 @@ test-all-in-one: ## Test docker images locally
 	&& $(MAKE) satellite-image storagenode-image gateway-image \
 	&& ./scripts/test-aio.sh
 
+.PHONY: test-sequential
+test-sequential: ## Run tests five times with caching turned off  and failfast
+	seq 1 5 | xargs -I{} sh -c 'go test -v ./... -count=1 -race -failfast'
```

Comment:
why can't you use -count=5 instead?

Reply:
You are 100% right, I'll update this to take that.